npm/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "stepsecuritynodetest",
```

Comment:
please publish test packages under a scope in the future; i see https://www.npmjs.com/package/stepsecuritynodetest already exists so the damage is done, but wanted to note that for the future.

Reply:
I will merge the changes and then update the name of the package in a future change. It could be something like `exfiltration-simulator`.

Reply:
I am leaning towards `malware-simulator` as the name learning from the name used here: https://github.com/qpwo/actual-malware

I could use a scope, e.g. `@step-security/malware-simulator`.